pkgs/development/python-modules/mypy/default.nix

```diff
@@ -24,8 +30,31 @@ buildPythonPackage rec {
     "mypy.api"
     "mypy.fastparse"
     "mypy.report"
+    "mypyc"
+    "mypyc.analysis"
+  ];
+
+  # These three patches are required to make compilation with mypyc work for
+  # 0.790, see also https://github.com/python/mypy/issues/9584.
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/python/mypy/commit/f6522ae646a8d87ce10549f29fcf961dc014f154.patch";
+      sha256 = "0d3jp4d0b7vdc0prk07grhajsy7x3wcynn2xysnszawiww93bfrh";
+    })
+    (fetchpatch {
+      url = "https://github.com/python/mypy/commit/acd603496237a78b109ca9d89991539633cbbb99.patch";
+      sha256 = "0ry1rxpz2ws7zzrmq09pra9dlzxb84zhs8kxwf5xii1k1bgmrljr";
+    })
+    (fetchpatch {
+      url = "https://github.com/python/mypy/commit/81818b23b5d53f31caf3515d6f0b54e3c018d790.patch";
+      sha256 = "002y24kfscywkw4mz9lndsps543j4xhr2kcnfbrqr4i0yxlvdbca";
+    })
   ];
 
+  # Compile mypy with mypyc, which makes mypy about 4 times faster. The compiled
+  # version is also the default in the wheels on Pypi that include binaries.
+  MYPY_USE_MYPYC = "1";
```

Comment:
Can we move this variable in the build or prebuild phase?

Reply:
How can I do that?